.gitpod.yml

```diff
@@ -32,3 +32,8 @@ github:
     addComment: false
     # add a "Review in Gitpod" button to the pull request's description (defaults to false)
     addBadge: true
+tasks:
+  - name: Dev Server main-site
+    init: npm install
+    command: npm run dev --workspace sites/main-site
```

Comment:
```suggestion
```
It usually is another subsite on git pod because you get main-site from the netlify deploy

Reply:
And move it maybe to the top where the commented out task is